rules/selector.js

```diff
@@ -21,6 +21,18 @@ module.exports = {
   'selector-descendant-combinator-no-non-space': true,
   // Specify a pattern for id selectors.
   'selector-id-pattern': /^[A-Z][a-zA-Z]+$/,
+  // Limit the number of attribute selectors in a selector. <FEEDBACK>
+  'selector-max-attribute': null,
+  // Limit the number of classes in a selector. <FEEDBACK>
+  'selector-max-class': null,
+  // Limit the number of combinators in a selector. <FEEDBACK>
+  'selector-max-combinators': null,
+  // Limit the number of id selectors in a selector. <FEEDBACK>
+  'selector-max-id': null,
+  // Limit the number of type in a selector. <FEEDBACK>
+  'selector-max-type': null,
+  // Limit the number of universal selectors in a selector. <FEEDBACK>
+  'selector-max-universal': 2,
```

Comment:
IMO (and I'd love some feedback from @Shopify/css):

* Max attributes: 1
* Max classes: 2
* Max combinators: 1
* Max ID: 0
* Max type: 0
* Max universal: 2

These are probably the strictest rules we can reliably adhere to, but I'm unsure if they are overly strict.